### PR TITLE
adding update_username function to change local username password

### PR DESCRIPTION
**Pull request by noredistribution**

### CODE REVIEW

File: cvprac/cvp_api.py [hunk 1]
```diff
@@ -96,6 +96,26 @@ def get_cvp_info(self):
             self.clnt.set_version(data['version'])
         return data
 
+    def update_username(self, username, password, role):
```
Comment (mharista) [inline cvprac/cvp_api.py:99]: The function name update_username seems a bit misleading. Isn't this function technically updating a given user accounts password and role? It isn't actually changing the username itself is it? If my assumption is right maybe the function name of update_user, or update_user_info, or update_user_data?

Comment (noredistribution) [reply cvprac/cvp_api.py:99]: yes you're right, we could go for `update_user` which is close to the name of the API (`updateUser.do`) or `update_user_info` ?

Comment (mharista) [reply cvprac/cvp_api.py:99]: I like update_user as it keeps the name shorter and closer to the API

File: cvprac/cvp_api.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#
+#F
```
Comment (mharista) [inline cvprac/cvp_api.py:1]: I'll remove this extra 'F' character.

File: test/system/test_cvp_api.py [hunk 3]
```diff
@@ -199,6 +199,34 @@ def test_api_get_cvp_info(self):
         self.assertEqual(self.clnt.version, result['version'])
         self.assertIsNotNone(self.clnt.apiversion)
 
+    def test_api_get_user(self):
```
Comment (mharista) [inline test/system/test_cvp_api.py:202]: I will consolidate these tests into one function.

**Pull request merged**